app/views/vm_common/_snapshots_tree.html.haml

```diff
@@ -2,17 +2,13 @@
   - if @record.snapshots.count > 0
     %fieldset
       %h3= _('Available Snapshots')
-      #snapshots_treebox{:style => "width: 100%"}
-      = render(:partial => "layouts/dynatree",
-        :locals => {:tree_id => "snapshots_treebox",
-          :tree_name => "snapshots_tree",
-          :id_prefix => "snap_",
-          :json_tree => @snaps,
-          :onclick => 'miqOnClickSnapshotTree',
-          :click_url => "/#{request.parameters["controller"]}/snap_pressed/",
-          :select_node => "#{session[:snap_selected]}",
-          :icon_size => 24,
-          :open_close_all_on_dbl_click => true})
-
+      = render(:partial => 'shared/tree', :locals => {:tree => @snapshot_tree, :name => @snapshot_tree.name})
+    :scss
+      .snapshot-active-suffix a {
+      background-color: #40a0ce !important;
+      color: white !important; /* @ IE6 */}
+      .snapshot-active-suffix a:after {
+      content: "#{_(' (Active)')}";}
```

Comment:
this looks really crazy `#{_('...` but is probably correct

I cannot find any decent documentation for the `:scss` HAML filter :-(